### DISCUSSION TITLE
Common tables - internal DB

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#10941] [Ideas]
Having tables that can be accessed by all apps would be incredibly helpful. This is not a new concern, as it has been mentioned by myself and others. It would be advantageous to have 2 or 3 tables, similar to the users table, that could be universally available. The employees table serves as an illustrative example of the kind of table I'm are referring to.

**Comment by melohagan** [comment]
Hey @JCV67 

Could be nice! Out of curiosity would you like the option to have universally available tables in addition to tables that can only be accessed on a per app basis, or would you be content if all tables were universally available in every app. 

Possibly in a similar way that users can be assigned access to an app, perhaps data tables could be assigned to apps as well.

**Comment by JCV67** [comment]
Helo @melohagan, thank you for replying 
I think that the option to have universally available tables in addition to tables that can only be accessed on a per app basis would be a good option.
An example: Workers table would be maintained by HR App, every new worker would be inserted there. The Work Planning App would access this table to form teams, choose team leaders, and so on. The Health an Safety App would also use this table schedule medical appointments, etc. Different relationship columns would be established from different tables in different Apps.

**Reply by JCV67** [reply]
This way we could build an entire ERP using only internal tables :-)

**Comment by poirazis** [comment]
Any update on this ?

**Comment by maciej-szulc-robotics** [comment]
Still not possible to share data between apps? Can I attach existing database (Couch DB) to another App?

**Reply by melohagan** [reply]
You could use the public API: https://docs.budibase.com/reference/rowsearch

Otherwise there is currently not a direct way to share the internal DB tables between apps. Maybe in this case one of the external databases would work best.
